test/Resources/LaminasModuleWithTraversableConfig.php

```diff
@@ -2,22 +2,19 @@
 
 /**
  * @see       https://github.com/laminas/laminas-config-aggregator-modulemanager for the canonical source repository
- * @copyright https://github.com/laminas/laminas-config-aggregator-modulemanager/blob/master/COPYRIGHT.md
- * @license   https://github.com/laminas/laminas-config-aggregator-modulemanager/blob/master/LICENSE.md New BSD License
  */
 
 declare(strict_types=1);
 
 namespace LaminasTest\ConfigAggregatorModuleManager\Resources;
 
 use ArrayObject;
-use stdClass;
 
 class LaminasModuleWithTraversableConfig
 {
     use ServiceManagerConfigurationTrait;
 
-    public function getConfig()
+    public function getConfig(): ArrayObject
```

Comment:
BC break

Reply:
Yes, you are right. But know, "cs-check" is broken.

Reply:
It is not:
> does not have return type hint nor *`@return` annotation* for its return value.

It wants this:
```php
/**
 * @return ArrayObject
 */
public function getConfig()